xlive/H2MOD/Modules/Shell/Startup/WinMainH2.cpp

```diff
@@ -94,6 +92,9 @@ int WINAPI WinMain_Halo2(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpC
 	// mess around with xlive (not calling XLiveInitialize etc)
 	WriteValue<BYTE>(Memory::GetAddress(0x4FAD98), 1);
 
+	// Apply patches for the hud that need to be applied before WinMain is called
+	hud_apply_pre_winmain_patches();
```

Comment:
move to tweaks, there patches are applied before startup

Reply:
moved it to tweaks